tour_model_eval/label_processing.py

```diff
@@ -89,6 +101,10 @@ def group_similar_trips(new_labels,track):
 
 
 # replace the first round labels with new labels
+# - track: a list to store the indices and labels from the first round of clustering
+# for item in track, item[0] is the original index of the trip in filter_trips
+# item[1] is the label after the first round of clustering
+# we change the labels from the first round with new labels from the second round here
```

Comment:
If this gets more complicated, it may be worthwhile to create a new class called `track` so this is encapsulated better.